### ISSUE TITLE
Batch Import-Interface

### ISSUE THREAD

**Issue by FlorianLautenschlager** [ChronixDB/prometheus#2]
Prometheus should provide a batch import interface. Extend the existing API.

Discuss:
 - Serialization of the batch: JSON vs. ProtoBuf vs. "something new" (Should be lightweight in terms of prometheus integration)

**Comment by juliusv** [comment]
At the moment it looks like this will not make it into upstream without a bigger debate (and then it's questionable), but it's easy enough to provide as a patch. Suggestion: I'll build the patch first and then we can decide whether it makes sense to drive a longer discussion upstream.

What was the exact use case for this again? AFAIR this was only for testing purposes (loading some predetermined set of test or benchmark data in) anyways, right?

**Comment by FlorianLautenschlager** [comment]
You are right. The use case is that we want a way to insert data into prometheus for testing purposes.
However we should discuss if we actually want/need this.
